package/libyui-qt.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Sun Feb  5 14:43:49 CET 2017 - besser82@fedoraproject.org
+
+- Fix crash when shutting down the UI (gh#libyui-libyui-qt#41)
```

Comment:
thanks, please add also reference to bugzilla, so we can link it and auto-submit it to suse. Thanks ( just add there `, bsc#931762)` then I will merge it.